prototype_source/fx_graph_mode_quant_guide.rst

```diff
@@ -126,8 +126,9 @@ quantization code:
       "non_traceable_module_class": [MNonTraceable],
   }
   model_prepared = prepare_fx(
-      model_fp32, 
-      qconfig_dict,
+      model_fp32,
+      qconfig_mapping,
+      example_inputs,
```

Comment:
do we want to define an `example_inputs` here?

Reply:
we don't define the other things so i think its fine